server/service/src/processors/central_records/mod.rs

```diff
@@ -74,8 +88,27 @@ pub(crate) fn process_central_records(
     Ok(())
 }
 
-trait ContactFormProcessor {
+pub(super) trait CentralServerProcessor {
     fn get_description(&self) -> String;
+    /// Default to using change_log_table_names
+    fn changelogs_filter(&self) -> ChangelogFilter {
+        ChangelogFilter::new().table_name(EqualFilter {
+            equal_any: Some(self.change_log_table_names()),
+            ..Default::default()
+        })
+    }
+
+    /// Default to empty array in case chanelogs_filter is manually implemented
+    fn change_log_table_names(&self) -> Vec<ChangelogTableName> {
+        Vec::new()
+    }
+
+    /// Extra check to see if processor should trigger, like if it's central for contact form email
+    fn should_run(&self) -> bool {
+        true
+    }
```

Comment:
If it's the central server processor, shouldn't it always only run on central server?